x/evm/types/precompiles.go

```diff
@@ -0,0 +1,28 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package types
+
+const (
+	P256PrecompileAddress         = "0x0000000000000000000000000000000000000100"
+	Bech32PrecompileAddress       = "0x0000000000000000000000000000000000000400"
+	StakingPrecompileAddress      = "0x0000000000000000000000000000000000000800"
+	DistributionPrecompileAddress = "0x0000000000000000000000000000000000000801"
+	ICS20PrecompileAddress        = "0x0000000000000000000000000000000000000802"
+	VestingPrecompileAddress      = "0x0000000000000000000000000000000000000803"
+	BankPrecompileAddress         = "0x0000000000000000000000000000000000000804"
+)
```

Comment:
nit: in line with opcodes constant identifiers, it would be nice to separate them to make more clear their namespace range:

```suggestion
const (
	P256PrecompileAddress         = "0x0000000000000000000000000000000000000100"
	Bech32PrecompileAddress       = "0x0000000000000000000000000000000000000400"
)

const = (
	StakingPrecompileAddress      = "0x0000000000000000000000000000000000000800"
	DistributionPrecompileAddress = "0x0000000000000000000000000000000000000801"
	ICS20PrecompileAddress        = "0x0000000000000000000000000000000000000802"
	VestingPrecompileAddress      = "0x0000000000000000000000000000000000000803"
	BankPrecompileAddress         = "0x0000000000000000000000000000000000000804"
)
```

Reply:
sure sounds good